extensions/wikia/SpecialVideos/SpecialVideosHelper.class.php

```diff
@@ -46,69 +46,17 @@ public function getMetaTagDescription() {
 	}
 
 	/**
-	 * get list of sorting options
-	 * @return array $options
-	 */
-	public function getSortOptions() {
-		$options = $this->getSortOptionsMobile();
-		$options['popular'] = wfMessage( 'specialvideos-sort-most-popular' )->plain();
-
-		return $options;
-	}
-
-	/**
-	 * get list of sorting options for mobile
-	 * @return array $options
-	 */
-	public function getSortOptionsMobile() {
-		$options = array(
-			'trend'   => wfMessage( 'specialvideos-sort-trending' )->plain(),
-			'recent'  => wfMessage( 'specialvideos-sort-latest' )->plain(),
-		);
-
-		return $options;
-	}
-
-	/**
-	 * get list of filter options
-	 * @return array $options
-	 */
-	public function getFilterOptions() {
-		$options = array();
-
-		$premiumVideos = $this->premiumVideosExist();
-		if ( !empty( $premiumVideos ) ) {
-			$options['premium'] = wfMessage( 'specialvideos-sort-featured' )->text();
-		}
-
-		if ( $this->wg->UseVideoVerticalFilters ) {
-			$options['trend:Games'] = wfMessage( 'specialvideos-filter-games' )->text();
-			$options['trend:Lifestyle'] = wfMessage( 'specialvideos-filter-lifestyle' )->text();
-			$options['trend:Entertainment'] = wfMessage( 'specialvideos-filter-entertainment' )->text();
-		}
-
-		return $options;
-	}
-
-	/**
 	 * get list of videos
-	 * @param string $sort [recent/popular/trend]
 	 * @param integer $page
+	 * @param string $filter [all/premium]
 	 * @param array $providers
 	 * @param string $category
 	 * @param array $options
 	 * @return array $videos
 	 */
-	public function getVideos( $sort, $page, $providers = [], $category = '', $options = [] ) {
```

Comment:
Did you check usage of this method. We are breaking backwards compatibility here.